blocks/init/src/blocks/components/header/header-style.scss

```diff
@@ -1,3 +1,68 @@
-// .header {
+$header: (
+  height: 50px,
+  site-padding: 60px,
+  site-container-wide: 1000px,
+  colors: (
+    background: #FFFFFF,
+  ),
+);
 
-// }
+.header {
+  $this: &;
+  padding: 0 map-get-deep($header, site-padding);
+  top: 0;
+  background: map-get-deep($header, colors, background);
+  height: map-get-deep($header, height);
+  width: 100%;
+  position: relative;
+  z-index: 100;
+
+  &--fixed {
+    position: fixed;
+  }
+
+  &__wrapper {
+    max-width: map-get-deep($header, site-container-wide);
+    margin: 0 auto;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__column {
+    flex: 1 0 auto;
+    height: 100%;
+    position: relative;
+    z-index: 100;
+
+    &--center {
+      text-align: center;
+    }
+
+    &--right {
+      text-align: right;
+    }
+
+    @include media(desktop up) {
+      &--left {
+        order: 2;
+      }
+
+      &--center {
+        text-align: left;
+        order: 1;
+      }
+
+      &--right {
+        order: 3;
+      }
+    }
+  }
+
+  .custom-logo {
```

Comment:
why is this out of bem

Reply:
Because default implementation (as I've used it) is injecting WordPress's native `get_custom_logo()` as the "Component" - which automatically adds the `custom-logo` class to the logo `<img>` (saved me having to write a logo component).

Reply:
The mockup that you provided should not have get_custom_logo() method at all. This is an example only and you should provide only minimal stuff. 
Remove the has_custom_logo and get_custom_logo from the implementation and create a new component called logo with link and image and provide this to the mockup.

This way that component is fully decoupled

Reply:
this still remains

Reply:
> this still remains

Overlooked this comment the first time because I've provided an explanation why this is implemented in such way.

> Remove the has_custom_logo and get_custom_logo from the implementation and create a new component called logo with link and image and provide this to the mockup.

I think that's overcomplicating things, we don't need a component for that. If I make it as dumb as it should be then it's just an "extended" `Image` component with 2 lines of CSS. If I don't then it's coupled to WordPress's functions (`get_custom_logo()`).

Reply:
This of this as investment for the future. In general you want to use logo on multiple places per web. This way you can easily call logo component and you are done.  You don't need to call manifest, etc. 

Also you can provide project home url link also. 
Logo is in general used on footer and header any maybe on some other places on the project. It makes sence to have it as a component. 

Also loading header then look soo clean. 

One more thing is that we had multiple logo images on the HPB, Infinum web that makes loading image soo simple if it is a component.

Please don't be afraid to use components. ;)